config/helper.go

```diff
@@ -0,0 +1,39 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func defaultTopicNameGenerator(group, topic, prefix string) string {
+	return fmt.Sprintf("%s.%s.%s", prefix, group, topic)
+}
+
+func envVarAsBool(name string) bool {
+	truthy := map[string]bool{
+		"true": true,
+		"1":    true,
+		"on":   true,
+	}
+
+	return truthy[strings.ToLower(os.Getenv(name))]
+}
+
+func envVarAsIntSlice(name string) ([]int, error) {
+	envVal := os.Getenv(name)
+	if envVal == "" {
+		return []int{}, nil
+	}
+
+	var i []int
+	for _, pt := range strings.Split(envVal, ",") {
+		val, err := strconv.Atoi(pt)
+		if err != nil {
+			return nil, err
+		}
+		i = append(i, val)
+	}
+	return i, nil
+}
```

Comment:
could you add some tests for this helper methods please?

Reply:
they are covered by the config tests already?